src/mdbook.rs

```diff
@@ -0,0 +1,15 @@
+pub fn handle_files(
+    config: &crate::Config,
+    files: &[crate::FileMode],
+) -> crate::Result<Vec<ft_api::bulk_update::Action>> {
+    let mdbook_config = mdbook::config::Config::default();
```

Comment:
We will have to make another config, `mdbook_config: String`, and use toml library to read it instead of creating default here.